example-new-architecture/android/build.gradle

```diff
@@ -3,11 +3,11 @@
 buildscript {
     ext {
         buildToolsVersion = "34.0.0"
-        minSdkVersion = 21
+        minSdkVersion = 23
```

Comment:
why do we raise min SDK? Is it because of the RN version used in this sample app?